### ISSUE TITLE
XP rate bug

### ISSUE THREAD

**Issue by Zeehra** [gamefreedomgit/Maelstrom#4078]
[//]: # (REMBEMBER! Add links to things related to the bug using for example:)
[//]: # (http://wowhead.com/)
[//]: # (cata-twinhead.twinstar.cz)

**Description:**
I am leveling a character on an account with 2 lvl85s, and should be getting an xp buff. However leveling seemed slow so I made sure to .xp reset. It showed me that my xp rate was at 3. I tried to .xp set 5 and .xp set 9 just to max it out, but it still only said 3. I asked in the discord support channel and was told to .xp reset. I then, as advised, made an in game ticket, and was again told to .xp reset. 

.xp reset does not seem to work. Maybe it's just a visual bug, but it feels slow af - I should be leveling way faster with a lvl25 guild and full looms.

**How to reproduce:**
I don't know to be frank.

**How it should work:**
I should be able to set my xp rate higher.

**Database links:**

https://imgur.com/a/hC4IhM0

**Comment by UltraNix** [comment]
@Zeehra Still a problem?

**Comment by Zeehra** [comment]
Yep, but not sure if it's just a visual bug. The GM I spoke to just said my xp rate was what it was supposed to be, but .xp display still just shows the base 3x and not the 3+whatever bonus is on that account.

**Comment by UltraNix** [comment]
@Zeehra And can you just check how much exp do you get with different settings?

**Comment by Zeehra** [comment]
> @Zeehra And can you just check how much exp do you get with different settings?

I'm not sure what you mean with "different settings", but I tried a new character on an account with 2 lvl85s. The character is level7, and fighting a lvl3 mob earned me 78 xp - no matter if I .xp reset, .xp set 9 (which defaults highest possible) or specifically .xp set 3. I tried setting my xp to 2, and got 52 xp per mob, so that ads up.

So I am not getting more than 3, despite having several 85'ers. Thought it was a visual bug, but I guess not.
